### ISSUE TITLE
ugly InsideDiamond with long conditions

### ISSUE THREAD

**Issue by DPDmancul** [plantuml/plantuml#649]
When the condition text is long the shape of the InnerDiamond is not padded enough; moreover the Padding doesn't work with Diamonds:
![Result](http://www.plantuml.com/plantuml/png/TL31QiCm3BthAtHhSjfWjxQCRXs3tGRR3vZOoKGSgLXAInZzzwbUUziC1htdztZFsgc5OlEKOQksvFG8JvtHaMnveJ2KC7q3uetosSyAxOiKJr96hSHxY95um8TxXsU_Rfs4up_ovz2DGv6Purwo55nIpdBwrLPfbCIwichvA-0CsxjhMT2H-12y9-uzaOo4FovF-CPACLsR0DHl0lLDJUXkMdHmJ6N1B3oq5zHyjbuuCU3QofmsaB8cR5vR9_hWipDkxc4TPGDgSe2T4xx17m00)
```puml
@startuml
<style>
ActivityDiagram{
  Activity{
    ' this works
    Padding 30
  }
  Diamond{
    ' this works
    BackgroundColor yellow
    ' this doens't work
    Padding 30
  }
}
</style>

skinparam ConditionStyle InsideDiamond

start
if(this condition is very long) is (T) then
   :true;
else (F)
   :false;
endif
:do;
stop
@enduml
```

**Comment by The-Lum** [comment]
Hello @DPDmancul,

A possible workaround is to add `\n\t`:

```puml
@startuml
skinparam ConditionStyle InsideDiamond
start
if(\n\tthis condition is very long\t\n) is (T) then
   :true;
else (F)
   :false;
endif
:do;
stop
@enduml
```

[![](http://www.plantuml.com/plantuml/svg/DSungiCm3CRnNQVGc5mXMGBlKUZSZbvCBJUYjXmifP3RLmrq_7xy4LhKOhUz5j0NonPxhFZN9B5naviTXV0goed-ELPtE7lWF0G9PYihFduv-dXJFx0qUGOBCdvbk8zeAmaWucHzfnceAE5m6K_AqPUR7yamfJQ3MjjWSV2tFW00)](http://www.plantuml.com/plantuml/uml/DSungiCm3CRnNQVGc5mXMGBlKUZSZbvCBJUYjXmifP3RLmrq_7xy4LhKOhUz5j0NonPxhFZN9B5naviTXV0goed-ELPtE7lWF0G9PYihFduv-dXJFx0qUGOBCdvbk8zeAmaWucHzfnceAE5m6K_AqPUR7yamfJQ3MjjWSV2tFW00)

Regards.

**Comment by DPDmancul** [comment]
Yes, this is the solution I'm actually using as a temporary fix.
The solution with the Padding in style would be obviously better, and a lot more better an automatic padding for long content, but I don't know if the latter could be easily feasible
